packages/autocomplete-core/src/types/api.ts

```diff
@@ -243,6 +266,10 @@ export interface PublicAutocompleteOptions<TItem> {
    * updating the state.
    */
   onInput?(params: OnInputParams<TItem>): void;
+  /**
+   * The array of plugins.
+   */
+  plugins?: Array<AutocompletePlugin<TItem, unknown>>;
```

Comment:
This must be `unknown`, right? We cannot know what it's going to be at this point. What do you think?

Reply:
Yep!